src/shared/views/Project.tsx

```diff
@@ -161,8 +162,16 @@ const ProjectView: React.FunctionComponent<ProjectViewProps> = ({
                       <a onClick={() => goToOrg(org.label)}>{org.label}</a> |{' '}
                     </span>
                   )}{' '}
-                  {project.label}{' '}
+                  {project.label}
+                  {'  '}
                 </h1>
+                <div style={{ marginLeft: 10 }}>
+                  <ViewStatisticsProgress
+                    orgLabel={(org && org.label) || ''}
+                    projectLabel={project.label}
+                    resourceId="nxv:defaultElasticSearchIndex"
```

Comment:
Pls use the constant we have for this id in another file, instead of a magic string

Reply:
we don't have an exported constant :man_shrugging: 

I'll create one

Reply:
here we go 16c8a07b7c7e57f6ccc9311ab1c7afef398f460d

Reply:
https://github.com/BlueBrain/nexus-sdk-js/pull/186